src/site/content/en/blog/codelab-address-form-best-practices/index.md

```diff
@@ -0,0 +1,296 @@
+  <input id="postal-code" name="postal-code" 
+    autocomplete="postal-code" maxlength="20">
+</section>
+
+<section id="country-region">
+  <label for="">Country or region</label>
+  <select id="country" name="country" autocomplete="country" 
+    required>
+      <option selected value="SPACER"> </option>
+      <option value="AF">Afghanistan</option>
+      <option value="AX">Åland Islands</option>
+      <option value="AL">Albania</option>
+      <option value="DZ">Algeria</option>
+      <option value="AS">American Samoa</option>
+  </select>  
+</section>
+```
+
+You'll see that **Postal code** is optional: that's because [many countries don't use postal codes](https://hellowahab.wordpress.com/2011/05/24/list-of-countries-without-postal-codes/). 
+([Global Sourcebook](https://www.grcdi.nl/gsb/global%20sourcebook.html) provides information about 
+address formats for 194 different countries, including sample addresses.) The label **Country or 
+region**  is used instead of **Country**, because some options from the full list (such as the 
+United Kingdom) are not single countries (despite the `autocomplete` value).
+
+## Step 4: Enable customers to easily enter shipping and billing addresses
+
+You've built a highly functional address form, but what if your site requires more than one 
+address, for shipping and billing, for example? Try updating your form to enable customers to enter 
+shipping and billing addresses. How can you make data entry as quick and easy as possible, 
+especially if the two addresses are the same? The article that goes with this codelab explains 
+[techniques for handling multiple addresses](/payment-and-address-form-best-practices#billing-address). 
+Whatever you do, make sure to use the correct `autocomplete` values!
+
+## Step 5: Add a telephone number field
+To add a telephone number input, add the following code to the form:
+
+```html
+<section>
+  <label for="tel">Telephone</label>
+  <input id="tel" name="tel" autocomplete="tel" type="tel" 
+    maxlength="30" pattern="[\d \-\+]+" enterkeyhint="done" 
+    required>
+</section>
+```
+
+For phone numbers use a single input: don't split the number into parts. That makes it easier for 
+users to enter data or copy and paste, makes validation simpler, and enables browsers to autofill. 
+
+There are two attributes that can improve the user experience of entering a telephone number: 
+* `type="tel"` ensures mobile users get the right keyboard.
+* `enterkeyhint="done"` sets the mobile keyboard enter key label to show that this is the last 
+field and the form can now be submitted (the default is `next`).
+
+<figure class="w-figure">
+  <img src="images/enterkeyhint.png" alt="Two screenshots of a form 
+  on Android showing how the enterkeyhint input attribute changes the enter key button icon.">
+  <figcaption class="w-figcaption">Use the enterkeyhint attribute to set the Enter button label: 'next' and 'done'.</figcaption>
+</figure>
+
+{% Aside 'gotcha' %}
```

Comment:
This not rendering as an aside, not sure why. A typo I'm not registering?

Reply:
Same for me, but rendered after a reload.

Reply:
I'd used `'gotcha'` not `'gotchas'`. Now fixed.